modules/vstudio/vs2005_dotnetbase.lua

```diff
@@ -820,6 +820,30 @@
 		end
 	end
 
+	function dotnetbase.netcore.getsdk(cfg)
+		if cfg.dotnetsdk == "web" then
+			return "Microsoft.NET.Sdk.Web"
+		elseif cfg.dotnetsdk == "razor" then
+			return "Microsoft.NET.Sdk.Razor"
+		elseif cfg.dotnetsdk == "worker" then
+			return "Microsoft.NET.Sdk.Worker"
+		elseif cfg.dotnetsdk == "blazor" then
+			return "Microsoft.NET.Sdk.BlazorWebAssembly"
+		elseif cfg.dotnetsdk ==  "windowsdesktop" then
+			return "Microsoft.NET.Sdk.WindowsDesktop"
+		elseif cfg.dotnetsdk == "mstest" then
+			return "MSTest.Sdk"
+		elseif not cfg.dotnetsdk then
+			return "Microsoft.NET.Sdk"
+		end
+	end
+
+	function dotnetbase.netcore.dotnetsdk(cfg)
+		if cfg.dotnetsdk == "mstest" then
+			_p(2,'')
```

Comment:
Is it intended to print nothing?

Reply:
i forgot to remove this! this was a remnent of trying to get the version fot mstest sdk specified it does not work when it is not specified! but that issue got fixed by using global.json